components/form/Footer.vue

```diff
@@ -60,8 +68,19 @@ export default {
       </slot>
       <slot name="middle" />
       <slot name="save">
-        <AsyncButton v-if="isEdit" key="edit" mode="edit" @click="save" />
-        <AsyncButton v-if="isCreate" key="create" mode="create" @click="save" />
+        <AsyncButton
+          v-if="isEdit"
+          key="edit"
+          mode="edit"
+          @click="save"
+        />
+        <AsyncButton
+          v-if="isCreate"
+          key="create"
+          mode="create"
+          :use-enable-label="useEnableLabelForCreate"
+          @click="save"
+        />
```

Comment:
3) Then get rid of both of these and do a single `<AsyncButton v-if="!isView" :mode="mode" @click="save"/>` to pass down and show the right label.